resolwe/process/__main__.py

```diff
@@ -95,13 +96,24 @@
         print("ERROR: Requirements failed to load: {}".format(str(error)))
         sys.exit(1)
 
+    # Prepare relations.
+    relations = None
+    try:
+        if args.relations:
```

Comment:
If this `if` needed as the parameter is required?

Reply:
Currently, this parameter is always given, but it is not considered as required - AFAIK argparse considers parameters starting with `--` as optional and the one without `--` as mandatory.
[Link do related argparse docs.](https://docs.python.org/3/library/argparse.html#name-or-flags)
For safety (and consistency with `--requirements`) i think it is better to leave it as is.

Reply:
Interesting, I didn't know that.